src/client/containers/past-speakers.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react'
+
+import Title from '../components/cover/title'
+import Headline from '../components/base/headline'
+import SubHeadline from '../components/base/subheadline'
+import Strip from '../components/base/strip'
+import Container from '../components/base/container'
+
+import Row from '../components/speakers/row'
+import Talk from '../components/speakers/talk'
+
+import pastSpeakersData from '../../assets/pastSpeakers.json'
+
+export default class PastSpeakers extends Component {
```

Comment:
We're using a class on App as well. This enables us to eject and remove react-router, if we need to. Otherwise this wouldn't work with react-hot-loader correctly. It might've been fixed in its latest version, but that doesn't really matter since no props will be excessively passed to the highest component.